posthog/cdp/templates/customerio/template_customerio.py

```diff
@@ -0,0 +1,106 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+# Based off of https://customer.io/docs/api/track/#operation/entity
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    id="template-customerio",
+    name="Update persons in Customer.io",
+    description="Updates persons in Customer.io",
+    icon_url="/api/projects/@current/hog_functions/icon/?id=customer.io",
+    hog="""
+fn callCustomerIoApi(method, path, body) {
+    // TODO: Base64 encode the site_id and token
```

Comment:
`base64Encode` and `Decode` should be there?

Reply:
Nice - updated it